### DISCUSSION TITLE
[0주차] 프로세스 주소 공간

### DISCUSSION THREAD

**Discussion by tnwlsgk1002** [interview-study-woory/interview-study#1] [Topic]
# 학습 정리 링크
> 학습 정리한 링크를 공유해주세요. 블로그, 노션 등 상관 없음

| 이름 | 학습 정리 링크                 |
|:-----:|:----------------------------:|
| 이수진👂 |  https://github.com/interview-study-woory/interview-study/discussions/new?category=topic |
| 유호현🎤 | https://github.com/interview-study-woory/interview-study/discussions/new?category=topic |
| 조재우👂 |   https://github.com/interview-study-woory/interview-study/discussions/new?category=topic                           |
| 전도명👂 |    https://github.com/interview-study-woory/interview-study/discussions/new?category=topic                          |

# 목차
> 각 질문 목차
- [프로세스와 스레드의 Context Switching은 어떤 차이가 있나요?](https://github.com/interview-study-woory/interview-study/discussions/1#discussioncomment-4016072)

**Comment by swjw13** [comment]
프로세스와 쓰레드의 Context Switching은 어떤 차이가 있나요?
